.safety-policy.yml

```diff
@@ -207,6 +207,13 @@ security:
             reason: jupyterlab  issue in version 4.03, requires < 4.0.11 python >= 3.8
         64588:
             reason: jupyterlab  issue in version 4.03, requires < 4.0.11 python >= 3.8
+        65212:
```

Comment:
Another ignore is needed for issue 65278:
```
        65278:
            reason: Fixed cryptography version 42.0.0 requires Python>=3.7 and is used there
```

Reply:
Done